aspnetcore/security/authentication/social/additional-claims/samples/6.x/ClaimsSample/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -266,7 +266,7 @@ public async Task<IActionResult> OnPostConfirmationAsync(string returnUrl = null
                             return RedirectToPage("./RegisterConfirmation", new { Email = Input.Email });
                         }
 
-                        await _signInManager.SignInAsync(user, isPersistent: false, info.LoginProvider);
+                        await _signInManager.SignInAsync(user, props, info.LoginProvider);
```

Comment:
We want to set IsPersistent to false if we want the claims to be refreshed more often I think

Reply:
I did not change the logic of the sample, the original sample set `isPersistent` to `true` in the `AuthenticationProperties` object (just a few lines above the call to `SignInAsync`), so I kept it that way. I limit my scope to the fix at hand.

Reply:
Well SignInAsync is currently passing in isPersistent: false so that's why I'm saying its definitely a behavior change, you can change isPersistent to false above if we want to maintain the same behavior is all I'm saying

Reply:
I have no problem with that, it's just that base branch has it set to `true`. No matter, I will make the change.

Reply:
Resolved by 9323a0a44f1d0fe0dbbe5b2908f05bafca77bfa6